### ISSUE TITLE
Rerun parameterized test

### ISSUE THREAD

**Issue by geokh77** [junit-team/junit5#1628]
Hi
I need to rerun failed tests that are parameterized. But I didn't manage. The parametrization is preventing the rerun of failed tests.

In JUnit 4 I could use reluse and control the outcome the test very easy, but with JUnit 5 and parameterization I cannot do that anymore!!???

Any suggestions?

Cheers George

**Comment by sormuras** [comment]
Are you on the command line? Within an IDE? Which?

**Comment by geokh77** [comment]
Hi Christian 
Thanks for your fast reply.
We built a test automation framework, based och Java, Selenium, JUnit, Gradle, TeamCiry, etc. When we run the tests on TeamCity the test fails due to connection error in the test environment ( we have integrations to third party, and this integrations go down sometimes causing our tests to fail and then the integrations are fine and the tests are green again, so the environment is flaky and gives flaky tests)
With JUnit 4 we used rules and checked if error is because of a network connection the we would catch the error and rerun the test once again at once.
But this not possible with JUnit 5 and parameterized tests! Or did I miss something?
Cheers George
